src/Signal-Estimator-GUI/extra_options.cpp

```diff
@@ -0,0 +1,69 @@
+#include "extra_options.h"
```

Comment:
I suggest to make two changes to file layout:

* move all existing source files from `src` to `src/cli`
* move the new UI code from `src/Signal-Estimator-GUI` to `src/gui`

I think this way it would be a bit cleaner.

Reply:
Also, can we adjust Signal-Estimator-GUI project so that it install the resulting binary into `bin` in project root instead of `src`?